### DISCUSSION TITLE
Automatic deployment and problem with css

### DISCUSSION THREAD

**Discussion by rudolfbruder** [wire-elements/spotlight#70] [Q&A] [answered]
Hi,

I am trying this package out locally and it works flawlessly, however when we run automatic deployment on our server this is how it looks:

<img width="1536" alt="image" src="https://user-images.githubusercontent.com/56951790/154010382-5c97c7eb-7a88-4938-b6f3-326d73cd6264.png">

Showing the console takes whole screen like some css was not built. When I manually run npm run prod on the vps all is good. Dont you know what might be wrong? Other css and js in the project works just fine.

**Comment by PhiloNL** [comment, marked as answer]
It seems that CSS classes are purged. Can you try and whitelist `'./vendor/wire-elements/spotlight/**/*.blade.php',` in your Tailwind CSS config?

**Reply by rudolfbruder** [reply]
Thanks that worked out!
